tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>Probably just "Setup." I'm noticing some inconsistencies in formatting and punctuation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>